webrtc.html

```diff
@@ -3284,39 +3284,59 @@ <h2>Attributes</h2>
-                  <code><a>RTCPeerConnection</a></code> object, except that this
-                  MUST NOT <a>Update the negotiation-needed flag</a>,  and,
-                  provided this does not fail, proceed with createOffer()'s
-                  regular steps.
-                </p>
-                <p>
-                  In all other situations, it will be disregarded.
-                </p>
+                <p>Offer to receive audio even though no audio will be sent.
+                When this dictionary member is present as an option to <a href=
+                "#dom-rtcpeerconnection-createoffer">createOffer</a>, run the
+                <a>offer to receive</a> steps with a <var>mediaType</var> of
+                "audio" instead of the regular <a href=
+                "#dom-rtcpeerconnection-createoffer">createOffer</a> steps.</p>
               </dd>
               <dt><dfn>offerToReceiveVideo</dfn> of type <span class="idlMemberType"><a>boolean</a></span></dt>
               <dd>
-                <p>
-                  Whenever this is given a non-false value, and the
-                  <code><a>RTCPeerConnection</a></code> has no non-stopped
-                  "sendrecv" or "recvonly" video transceivers, createOffer()
-                  MUST as its first step invoke the equivalent of
-                  <code>addTransceiver("video")</code> on the
-                  <code><a>RTCPeerConnection</a></code> object, except that this
-                  MUST NOT <a>Update the negotiation-needed flag</a>,  and,
-                  provided this does not fail, proceed with createOffer()'s
-                  regular steps.
-                </p>
-                <p>
-                  In all other situations, it will be disregarded.
-                </p>
+                <p>Offer to receive video even though no video will be sent.
+                When this dictionary member is present as an option to <a href=
+                "#dom-rtcpeerconnection-createoffer">createOffer</a>, run the
+                <a>offer to receive</a> steps with a <var>mediaType</var> of
+                "video" instead of the regular <a href=
+                "#dom-rtcpeerconnection-createoffer">createOffer</a> steps.</p>
               </dd>
             </dl>
+            <p>To <dfn>offer to receive</dfn> with <var>mediaType</var>, run
+            the following steps:</p>
+            <ol>
+              <li>
+                <p>Let <var>connection</var> be the current <code>
+                <a>RTCPeerConnection</a></code> object.</p>
+              </li>
+              <li>
+                <p>If the value of the dictionary member is false, jump to the
+                step labeled <em>create offer</em>.</p>
+              </li>
+              <li>
+                <p>If <var>connection</var> has any stopped "sendrecv" or
+                "recvonly" transceivers of type <var>mediaType</var>, jump to
+                the step labeled <em>create offer</em>.</p>
```

Comment:
@adam-be This flips the logic from *"Whenever this ... has no* ***UN-stopped*** *... transceivers"*, to *"Whenever this ... has no* ***STOPPED*** *... transceivers"*. Seems wrong (out of scope for fixing https://github.com/w3c/webrtc-pc/issues/1640 at least).

Reply:
Found by running the [wpt test](https://github.com/w3c/web-platform-tests/pull/8450#issuecomment-348316922). 👍

Reply:
Good catch! Let's revert that language back to what it was.